Ft signup page 163348459

Fiyiin opened the pull request:
### What does this PR do?
This PR adds the user registration form to the application.

### How should this be manually tested?
- Visit https://fiyiin.github.io/politico/ui/index.html
- Click on the signup button
- Or go to Visit https://fiyiin.github.io/politico/ui/signup.html

### Any background context you want to provide
The signup page is where first-time users provide registration details so they can create an account

### What are the relevant Pivotal Tracker Stories
Story id: #163348459

### Questions
None

![up](https://user-images.githubusercontent.com/38258693/51660572-76d5bb80-1fae-11e9-8a28-a9486623c4bf.PNG)

Felglitzzz commented:
@Fiyiin 
- your design for the `signup` and `login page` seems to be different. is there a reason why it is so ?
- fix the link in your PR template

Fiyiin commented:
@Felglitzzz 

- There was no reason, I was in a hurry to get something up. Fixed( I think)
- Fixed the link.

![up](https://user-images.githubusercontent.com/38258693/51606315-db8f0880-1f11-11e9-98a9-5a63a0b85839.PNG)
![in](https://user-images.githubusercontent.com/38258693/51606318-dc279f00-1f11-11e9-89f1-34dfa1d1b156.PNG)

Felglitzzz commented:
@Fiyiin, link still not working
